README.md

```diff
@@ -100,12 +105,11 @@ Software vs DMS build version map
 Unit Tests
 ----------
 
-Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.  We also recommend using `pytest-xdist` so you can run them in parallel.
+Unit tests can be run via `pytest`.  All tests need a the `ci_watson` pytest plugin to run.
 
-    pip install ci-watson pytest-xdist
-    pytest -n <cores>
+    pip install requests_mock ci_watson
+    pytest jwst
```

Comment:
Removing the `pytest-xdist` example here, as this does not work for people who are not using the CRDS central store cache and need to get the files via the CRDS web service.